src/main.ts

```diff
@@ -3,6 +3,15 @@ import * as cache from './cache';
 
 export async function run() {
   try {
+    core.info('------------------------');
+    core.info(
+      'NOTE: this action is deprecated and will not be maintained in future by GitHub engineers.'
+    );
+    core.info(
+      'Please, migrate to https://github.com/ruby/setup-ruby from official Ruby community.'
```

Comment:
This sounds a bit odd to me because there is no "official Ruby community". I don't know if it really matters who is maintaining this new action either. This message is directed at people already using this action and you're making a recommendation to them about what to use instead. I can appreciate that indicating it's from the Ruby community can invoke trust, but the consumers of this project already trust you and likely will trust your recommendations.

I'd suggest something like:

> Please migrate to the "ruby/setup-ruby" action, which is being actively maintained.

If you want to keep the original text, please note that the sentence is missing an article. It should be "from **the** official Ruby community."

Reply:
I think "official Ruby community" could be "official Ruby organization".  Any repo's within the GitHub 'ruby' account would be considered 'official'?

Reply:
@MSP-Greg, I think yes, I updated wording and changed to "organization" instead of community